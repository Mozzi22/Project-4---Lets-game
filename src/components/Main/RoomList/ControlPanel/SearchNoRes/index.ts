import SearchNoRes from './SearchNoRes';

export default SearchNoRes;
