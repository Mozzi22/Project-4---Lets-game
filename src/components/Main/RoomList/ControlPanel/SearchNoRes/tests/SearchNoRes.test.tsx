import React from 'react';
import SearchNoRes from '../SearchNoRes';
import { mount, shallow } from 'enzyme';

describe('SearchNoRes', () => {
    it('Should match snapshot', () => {
        const component = shallow(<SearchNoRes/>);
        expect(component.html()).toMatchSnapshot();
    });
    it('should render StNoResWrapper', () => {
        const component = mount(<SearchNoRes/>);
        expect(component.find('styled__StNoResWrapper')).toHaveLength(1);
    });
 it('should render StNoResTitle', () => {
        const component = mount(<SearchNoRes/>);
        expect(component.find('styled__StNoResTitle')).toHaveLength(1);
    });
    it('should render StNoResText', () => {
        const component = mount(<SearchNoRes/>);
        expect(component.find('styled__StNoResText')).toHaveLength(1);
    });
});