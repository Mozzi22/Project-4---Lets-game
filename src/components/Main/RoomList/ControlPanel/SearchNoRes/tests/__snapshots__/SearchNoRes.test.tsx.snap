// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchNoRes Should match snapshot 1`] = `"<div class=\\"styled__StNoResWrapper-sc-dn15ik-0 pTUWO\\"><p class=\\"styled__StNoResTitle-sc-dn15ik-1 utHSV\\">no_result</p><span class=\\"styled__StNoResText-sc-dn15ik-2 gCJYCb\\">try_new</span></div>"`;
